### ISSUE TITLE
podman crashes with multi-stage and incorrect copy --from my-registry/my-image command

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [containers/buildah#1894]
(Moving this from: https://github.com/containers/libpod/issues/4190)
Hi team, great job,

I faced a crash with a malformed Dockerfile using multistage and:
```
COPY --from quay.io/openshift/origin-cli /usr/bin/oc /usr/bin/oc
```
The malformed command here, is an equal sign missing between from and image name:
it should be
```
COPY --from=quay.io/openshift/origin-cli /usr/bin/oc /usr/bin/oc
```

Here is the stacktrace
```
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x5654a26bfa00, 0x5654a31a3fe0)
	/usr/lib/golang/src/runtime/panic.go:556 +0x2cf fp=0xc0005aff28 sp=0xc0005afe98 pc=0x5654a1309e8f
runtime.panicindex()
	/usr/lib/golang/src/runtime/panic.go:44 +0xfc fp=0xc0005aff58 sp=0xc0005aff28 pc=0x5654a1308b7c
github.com/containers/libpod/vendor/github.com/containers/buildah/imagebuildah.(*Executor).Execute(0xc0001dc900, 0x5654a2842320, 0xc00003c108, 0x0, 0x5654a1fdc36f, 0x1, 0xc00025a2c0, 0xc000239e30, 0xc00024e660, 0x23)
	/builddir/build/BUILD/libpod-f60417506b0bd363749b0d32d6ffaced763c71be/_build/src/github.com/containers/libpod/vendor/github.com/containers/buildah/imagebuildah/build.go:873 +0x1aeb fp=0xc0005b02c0 sp=0xc0005aff58 pc=0x5654a1cd15eb
github.com/containers/libpod/vendor/github.com/containers/buildah/imagebuildah.(*Executor).Build(0xc0001dc600, 0x5654a2842320, 0xc00003c108, 0xc0004552c0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, ...)
	/builddir/build/BUILD/libpod-f60417506b0bd363749b0d32d6ffaced763c71be/_build/src/github.com/containers/libpod/vendor/github.com/containers/buildah/imagebuildah/build.go:1282 +0x6dd fp=0xc0005b04e8 sp=0xc0005b02c0 pc=0x5654a1cd5efd
github.com/containers/libpod/vendor/github.com/containers/buildah/imagebuildah.BuildDockerfiles(0x5654a2842320, 0xc00003c108, 0x5654a2853060, 0xc0001dafc0, 0xc000330120, 0x2d, 0x0, 0x0, 0x0, 0x0, ...)
	/builddir/build/BUILD/libpod-f60417506b0bd363749b0d32d6ffaced763c71be/_build/src/github.com/containers/libpod/vendor/github.com/containers/buildah/imagebuildah/build.go:1440 +0x646 fp=0xc0005b0988 sp=0xc0005b04e8 pc=0x5654a1cd6ad6
```

**Comment by TomSweeneyRedHat** [comment]
FYI @akram

**Comment by mhrivnak** [comment]
I hit the same bug with yet a different typo. Here's a reproducer:

```
FROM registry.svc.ci.openshift.org/openshift/release:golang-1.11 AS builder
RUN touch foo

FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
COPY --from-builder /foo /foo
```

**Comment by TomSweeneyRedHat** [comment]
@m7mdkamal might this be one you could tackle?  Basically we need to make sure to avoid the stack and give a valid error if there's no equal sign involved with COPY's --from option.

**Comment by m7mdkamal** [comment]
@TomSweeneyRedHat Shouldn't it be fixed with #1898 
if there's no equal sign it won't split therefore with return an error!

**Comment by TomSweeneyRedHat** [comment]
@m7mdkamal Darn you're good!  That's gotta be the fastest bug fix ever for Buildah!  :1st_place_medal:  

You're right, it's working now:

```
# cat Dockerfile.withcopytom
FROM alpine as builder 
FROM fedora
COPY --from builder /bin/df /tmp/df_tom

# buildah bud -t tom -f Dockerfile.withcopytom .
STEP 1: FROM alpine AS builder
Getting image source signatures
Copying blob 89d9c30c1d48 done
Copying config 965ea09ff2 done
Writing manifest to image destination
Storing signatures
965ea09ff2ebd2b9eeec88cd822ce156f6674c7e99be082c7efac3c62f3ff652
STEP 2: FROM fedora
STEP 3: COPY --from builder /bin/df /tmp/df_tom
COPY: invalid --from flag, should be --from=<name|index>
```

Thank you very much for looking at this.  I, unfortunately, don't have a good first-time bug handy at the moment, but you're welcome to take any that hasn't been grabbed yet.
